src/core_ocean/mode_forward/mpas_ocn_forward_mode.F

```diff
@@ -495,6 +497,8 @@ function ocn_forward_mode_run(domain) result(ierr)!{{{
                                                    config_use_activeTracers_surface_restoring)
       call mpas_pool_get_config(domain % configs, 'config_use_surface_salinity_monthly_restoring',  &
                                                    config_use_surface_salinity_monthly_restoring)
+      call MPAS_pool_get_config(domain % configs, 'config_use_time_varying_atmospheric_forcing', &
+                                                   config_use_atmospheric_forcing)
```

Comment:
Best to use exactly the same name as the config flag. Otherwise searches can get confusing.  I can fix that.